src/_locales/en_US/messages.json

```diff
@@ -445,6 +445,14 @@
             }
         }
     },
+    "popup_info_replaced_widgets": {
+        "message": "One or more widgets have been replaced",
+        "description": "Clickable status header text in the popup. Shown when a media widget like Spotify embed or Disqus comment section has been temporarily replaced by Privacy Badger with a click-to-activate placeholder."
+    },
+    "popup_info_widgets_description": {
+        "message": "Some embedded content on this page might be used to track you, so <a href='https://privacybadger.org/#How-does-Privacy-Badger-handle-social-media-widgets' target='_blank'>Privacy Badger has temporarily replaced them</a>. The following widgets have been replaced:",
+        "description": "Clickable status header text in the popup. Shown when a media widget like Spotify embed or Disqus comment section has been temporarily replaced by Privacy Badger with a click-to-activate placeholder.",
```

Comment:
Chrome refuses to load the extension because of [invalid JSON](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas#trailing_commas_in_json).

Reply:
Fixed, thanks for pointing out